egs/mgb2/ASR/conformer_ctc/generate_unique_lexicon.py

```diff
@@ -0,0 +1 @@
+../../../librispeech/ASR/conformer_ctc/generate_unique_lexicon.py
```

Comment:
Also this one. It should be placed in `local`, not in `conformer_ctc`.

Reply:
Ok I fixed them

Reply:
@csukuangfj please let me know if there is anything else I need to do to merge the PR